src/wp-admin/about.php

```diff
@@ -50,187 +42,168 @@
+					<?php
+					_e( '<strong>Font-size adjustment in more places:</strong> now, font-size controls are right where you need them in the List and Code blocks. No more trekking to another screen to make that single change!' );
+					?>
+				</p>
+				<p>
+					<?php
+					_e( '<strong>Reusable blocks:</strong> several enhancements make reusable blocks more stable and easier to use. And now they save automatically with the post when you click the Update button.' );
+					?>
+				</p>
+				<p>
+					<?php
+					_e( '<strong>Inserter drag-and-drop:</strong> drag blocks and block patterns from the inserter right into your post.' );
+					?>
+				</p>
 			</div>
-		</div>
-
-		<div class="about__section">
-			<div class="column about__image is-edge-to-edge">
-				<img src="https://s.w.org/images/core/5.6/twentytwentyone-rainbow.png" alt="" />
+			<div class="column about__image">
+				<img src="data:image/svg+xml,%3Csvg width='269' height='120' xmlns='http://www.w3.org/2000/svg'%3E%3Crect x='0' y='0' width='100%25' height='100%25' fill='%23ddd' /%3E%3Ctext text-anchor='middle' font-family='sans-serif' font-size='24' y='50%25' x='50%25'%3EVideo%3C/text%3E%3C/svg%3E" alt="" />
 			</div>
 		</div>
 
-		<hr />
-		<hr class="is-large" />
-
-		<div class="about__section">
-			<header class="column is-edge-to-edge">
-				<h2><?php _e( 'Improvements for everyone' ); ?></h2>
-			</header>
-		</div>
-
-		<div class="about__section has-3-columns has-gutters">
-			<div class="column has-border" style="background-color:#eeeadd;background-color:var(--global--color-yellow)">
-				<h3><?php _e( 'Expanding auto-updates' ); ?></h3>
-				<p><?php _e( 'For years, only developers have been able to update WordPress automatically. But now, you have that option, right in your dashboard. If this is your first site, you have auto-updates ready to go, right now! Upgrading an existing site? No problem! Everything is the same as it was before.' ); ?></p>
-			</div>
-			<div class="column has-border" style="background-color:#e4d1d1;background-color:var(--global--color-red)">
-				<h3><?php _e( 'Accessibility statement template' ); ?></h3>
+		<div class="about__section has-2-columns">
+			<h2 class="is-section-header is-smaller-heading">
+				<?php _e( 'You can do more without writing custom code' ); ?>
+			</h2>
+			<div class="column">
 				<p>
 					<?php
-					printf(
-						/* translators: %s: Accessibility statement feature plugin link. */
-						__( 'Even if you’re not an expert, you can start letting folks know about your site’s commitment to accessibility at the click of a button! The new <a href="%s">feature plugin</a> includes template copy for you to update and publish, and it’s written to support different contexts and jurisdictions.' ),
-						'https://github.com/10degrees/accessibility-statement-plugin'
-					);
+					_e( '<strong>Full-height alignment:</strong> have you ever wanted to make a block, like the Cover block, fill the whole window? Now you can.' );
```

Comment:
```suggestion
					_e( '<strong>Full-height alignment:</strong> Have you ever wanted to make a block, like the Cover block, fill the whole window? Now you can.' );
```

Reply:
to match "social icons block"

Reply:
The rest of these are lowercase (including the section above this), should the Social Icons "You" instead be made lowercase?